web/src/lib/components/admin-page/settings/setting-switch.svelte

```diff
@@ -1,12 +1,19 @@
 <script lang="ts">
   import { quintOut } from 'svelte/easing';
   import { fly } from 'svelte/transition';
+  import { createEventDispatcher } from 'svelte';
 
   export let title: string;
   export let subtitle = '';
   export let checked = false;
   export let disabled = false;
   export let isEdited = false;
+
+  const dispatch = createEventDispatcher();
```

Comment:
```suggestion
  const dispatch = createEventDispatcher<{ toggle: boolean }>();
```

Best to keep these events typed for improved intellisense and type checking.